press/press/doctype/site_update/site_update.py

```diff
@@ -99,12 +99,32 @@ def validate_apps(self):
 		site_apps = [app.app for app in frappe.get_doc("Site", self.site).apps]
 		bench_apps = [app.app for app in frappe.get_doc("Bench", self.destination_bench).apps]
 
+		fields = ["old_name", "new_name"]
+		filters = { "old_name": ["in", site_apps] }
+		app_renames = frappe.get_list("App Rename Map", fields=fields, filters=filters)
+
+		for i in app_renames:
+			if i.old_name in site_apps:
+				site_apps.remove(i.old_name)
```

Comment:
I think there is some frappe.utils.find we can use here.

`i` feels weird tbh :sweat_smile: